lib/pagination/config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  defaultLimit: 25,
+  defaultPage: 1,
+  pageParam: 'page',
+  perPageParam: 'limit',
+  formatter: './paginate',
+  maxLimit: 50,
+  defaultAdapter: './adapter'
```

Comment:
This is used somehow ? Instead of passing a path for adapter use directly and object here (same for formatter)

Reply:
Right now, I'm just using defaultLimit and defaultParam, I made this to leave a base of configuration. For the Adapter and Formatter the idea is similar of what you say, but in the future the idea is that someone could create his own adapter, and change this configuration to point to that adapter. As now doesn't have any effect, I think is no necessary to change it yet